package.json

```diff
@@ -18,7 +18,8 @@
   ],
   "dependencies": {
     "bson": "~1.0.4",
-    "require_optional": "~1.0.0"
+    "require_optional": "~1.0.0",
+    "snappy": "~6.0.0"
```

Comment:
Move to devDependencies as it needs to be an optional dependency (all native libs are optional dependencies for the driver). This is critical for windows users that usually don't have a full compiler toolchain installed.